### DISCUSSION TITLE
Convergence Problem of "ConstantDT" for crystal plasticity model with finner mesh

### DISCUSSION THREAD

**Discussion by SuperkakaSCU** [idaholab/moose#26524] [Q&A Meshing]
Dear MOOSE experts,

The error occurs when I test the same CPFEM model with a finer mesh. For example, for 'substep.i', when I change the original 2x2x2 mesh to 4x4x4 mesh, the error message is shown below:

*** ERROR ***
The following error occurred in the object "ConstantDT", of type "ConstantDT".

Solve failed and timestep already at or below dtmin, cannot continue!

Similar issues happened to some other programs using Kalidindi's model. Would you happen to know the reason behind it? Can I fix it without decreasing the time step? Thanks!

**Comment by GiudGiud** [comment]
Hello

This is an error due to the solve failing, so the time step gets reduced (or attempts to) and goes below the allowed minimum. 
You can try to prevent the solve from failing, see this page:
https://mooseframework.inl.gov/moose/application_usage/failed_solves.html

Guillaume
